app/jobs/memberships/send_membership_request_mailer_job.rb

```diff
@@ -0,0 +1,21 @@
+#frozen_string_literal: true
+
+class SendMembershipRequestMailerJob < CaseflowJob
+  queue_with_priority :low_priority
+
+  def perform(email_type, recipient_info)
+    email_to_send(email_type, recipient_info).deliver_now!
+  end
+
+  private
+
+  def email_to_send
```

Comment:
```suggestion
  def email_to_send(email_type, recipient_info)
```

The parameter list needs to be added to the method signature for `email_to_send`.

Reply:
Updated, I understand why it has to be added. Thank you for catching this.